spec/lib/fastly_nsq/fake_message_queue_spec.rb

```diff
@@ -37,30 +37,19 @@
 end
 
 RSpec.describe FakeMessageQueue::Producer do
-  after do
-    FakeMessageQueue.reset!
-  end
+  after { FakeMessageQueue.reset! }
```

Comment:
I believe I set this in a `before` already in the `spec_helper` https://github.com/fastly/fastly_nsq/blob/master/spec/spec_helper.rb#L36